services/zwift_service.py

```
@@ -0,0 +1,72 @@
+            self.logger.info("No activities found on Zwift")
+            return None
+
+        last_activity = activities[0]  # The most recent activity
+        activity_id = last_activity['id']
+        self.logger.info(f"Downloading activity {activity_id}...")
+
+        link = f"https://{last_activity['fitFileBucket']}.s3.amazonaws.com/{last_activity['fitFileKey']}"
+        self.logger.info(f"Download link: {link}")
+
+        try:
+            response = requests.get(link)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)